joss wording (#27)
pyOpenSci/pyopensci.github.io: _pages/home.md

The [Journal of Open Source Software](https://joss.theoj.org/) is a community dedicated
to improving the visibility and quality of scientific software. They do

> so by providing a review and publishing process so that authors

Reviewer: ```suggestion
so by providing a review and publishing process similar to pyOpenSci so that authors
```

of packages can publish their packages with a DOI and citable artifact.

> JOSS reviews are limited in scope and the

Reviewer: ```suggestion
JOSS reviews are [more limited scope](https://joss.readthedocs.io/en/latest/review_criteria.html) compared to pyOpenSci and the
```

The [Journal of Open Source Software](https://joss.theoj.org/) is a community dedicated
to improving the visibility and quality of scientific software. They do
so by providing a review and publishing process similar to pyOpenSci so that authors
of packages can publish their packages with a DOI and citable artifact.
JOSS reviews are [more limited scope](https://joss.readthedocs.io/en/latest/review_criteria.html) compared to pyOpenSci and the
[submission criteria](https://joss.readthedocs.io/en/latest/review_criteria.html)

> are less stringent than those of pyOpenSci.

Reviewer: ```suggestion
are, in places, less stringent than those of pyOpenSci.
```
